packages/theme/stories/introduction.stories.mdx

```diff
@@ -15,35 +15,23 @@ import { CornerBlock } from "docs/components/CornerBlock";
+- While CSS-in-JS solutions have improved in terms of performance most still involve some runtime overhead, embracing vanilla CSS removes this overhead.
+- With vanilla CSS making up the foundation of a Toolkit theme, the theme itself can be published and consumed independently from the component libraries. It can be used in contexts where Toolkit components are not even used, e.g. static code deployment in CMS type systems.
+- Separation of styling into CSS files also reduces the tie-in with React, which might not be important in the short term, but longer term opens up the possibility that we could share styling assets with non-react components.
+  The pattern adopted by the UI Toolkit for styling provides not only the mechanism by which Toolkit components
+  themselves are styled, it also provides a consistent approach for styling all elements of your application. Most
+  applications will probably contain some mix of Toolkit components, plain HTML infrastructure, often for layout purposes,
+  custom components you build yourself, and maybe some third party components. You should be able to make use of the
+  styling primitives offered by the Toolkit theme to help style all of these. The goal is to make it as easy as possible to
+  achieve consistency of styling across your entire application.
 
-More importantly, with vanilla CSS making up the foundation of a Toolkit theme, the theme itself can be published and
-consumed independently from the component libraries. It can be used in contexts where Toolkit components are not even used,
-e.g. static code deployment in CMS type systems. Separation of styling into CSS files also reduces the tie-in with React,
-which might not be important in the short term, but longer term opens up the possibility that we could share styling assets
-with non-react components.
-
-#### Theme CSS + Component CSS
-
-The pattern adopted by the UI Toolkit for styling provides not only the mechanism by which Toolkit components
-themselves are styled, it also provides a consistent approach for styling all elements of your application. Most
-applications will probably contain some mix of Toolkit components, plain HTML infrastructure, often for layout purposes,
-custom components you build yourself, and maybe some third party components. You should be able to make use of the
-styling primitives offered by the Toolkit theme to help style all of these. The goal is to make it as easy as possible to
-achieve consistency of styling across the entire application.
-
-The entire solution is built around vanilla CSS (in other words, this is not a `CSS-in-JS` solution). The CSS comprises
-two kinds of declarations:
+The CSS comprises the following two kinds of declarations:
 
 - CSS Custom Properties declarations a.k.a. CSS Variables
```

Comment:
Nit: missing a full stop here.

Reply:
It's part of a list and not a full sentence

Reply:
There are two guidelines which apply here:

- Add a full stop **at the end of complete sentences**—or phrases that complete a sentence that the lead-in phrase begins (example below). - this is the one I followed as it isn't a complete sentence
- Format each bullet point the same way, i.e., write them all as phrases or all as complete sentences. - bearing this one in mind I will need to rewrite the first bullet to make it a complete sentence.

Thanks Josh